docs/actions.md

```diff
@@ -352,6 +352,79 @@ class MyTest {
 }
 ```
 
+### Verifying action registration
+
+A common mistake is implementing a `WebAction` but forgetting to register it via 
+`WebActionModule.create<YourAction>()`. Without registration, the action won't be exposed as an 
+HTTP endpoint.
+
+Misk provides `WebActionRegistrationTesting` to catch this. It scans your service's packages for 
+`WebAction` implementations and verifies each has a corresponding `WebActionEntry` registration.
+
+**Using the base class:**
+
+```kotlin
+class WebActionRegistrationTest : AbstractWebActionRegistrationTest() {
+  override fun createInjector(): Injector {
+    // Use your service's testing module
+    return Guice.createInjector(MyServiceTestingModule())
+  }
+
+  override fun webActionPackages(): List<String> {
+    // Scan your service's packages for WebAction implementations
+    return listOf("com.example.myservice")
+  }
+
+  // Optional: exclude specific actions from verification
+  override fun excludeWebAction(actionClass: KClass<out WebAction>): Boolean {
+    return actionClass.simpleName in setOf("TestOnlyAction", "SpecialAdminAction")
+  }
+
+  // Optional: hint shown in error messages
+  override fun registrationModuleHint(): String = "WebModule"
```

Comment:
```suggestion
  override fun registrationModuleHint(): String = "your service's web actions module"
```

if we don't know it we shouldn't put in a fake module in the hint that doesn't exist which would add to the confusion when someone tries to look for it